USAGE.md

```diff
@@ -187,6 +187,12 @@ For example, if we defined the following schema definition in a file named `mypr
         Unique identifier of the widget.
 ```
 
+Multiple directory targets can also be provided:
```

Comment:
Ah, I guess I was misunderstanding how to properly pass in multiple directories. I knew the `action=append` thing would make it possible to pass in the same flag multiple times, but I didn't realize we could just pass in space-separated paths as well.